frodo.js

```diff
@@ -2,10 +2,9 @@
  * Extras Plugin
```

Comment:
SCRIPT-I like how you've used promise chaining here, it makes the code flow very clear.